File: programmers/lv2/h-index.py
def solution(citations):
  citations.sort()
  hIndex = 0
  for i in range(1, citations[len(citations) - 1]):
    cnt = 0
    for j in citations:
      if j >= i:
        cnt += 1
    if cnt >= i:
      hIndex = i
    else:
      break
  return hIndex

File: programmers/lv2/h-index.js
function solution(citations) {
  var answer = 0;
  let maxValue = Math.max(...citations);

  for (let h = maxValue; h > -1; h--) {
    let cnt = 0;
    for (let i of citations) if (i >= h) cnt += 1;
    if (cnt >= h) answer = Math.max(answer, h);
  }

  return answer;
}
